keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -844,6 +858,11 @@ Yes, the feature may impact CreateVolume. We will measure this impact during bet
 
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
+Using this feature may result in non-negligible increase of resource usage IF customers batch modify many volumes at once and increase default external-resizer `--workers`, `-kube-api-burst`, and `-kube-api-qps` options.
```

Comment:
Are kube-api-burst and kube-api-qps options obsolete now that we have api server priority and fairness?

Reply:
Will remove mention of increasing these default parameters. However a batch volume modification may still lead to a spike in resource usage if CSI Controller is part of a `PriorityClass` with a higher priority (e.g. `system`), yes?